tests/virtctl/usbredir.go

```diff
@@ -161,23 +145,23 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 		It("Should work several times", func() {
 			for i := 0; i < 4*v1.UsbClientPassthroughMaxNumberOf; i++ {
```

Comment:
nit: You could use `range` in the other loops too

Reply:
ok